File: app/src/main/java/com/esq/drohealthtest/utils/StringUtils.java
package com.esq.drohealthtest.utils;

public class StringUtils {

    public static String getNumberOfItemsToString(int number) {
        if (number < 1) {
            return String.format("%d %s", 0, "item(s)");
        } else {
            return String.format("%d %s", number, "item(s)");
        }
    }

    public static String getPriceNumberToString(int number) {
        if (number < 1) {
            return String.format("%s%d", "₦", 0);
        } else {
            return String.format("%s%d", "₦", number);
        }
    }
}

File: app/src/main/java/com/esq/drohealthtest/utils/StringUtils.kt
package com.esq.drohealthtest.utils

object StringUtils {

    fun getNumberOfItemsToString(number: Int): String {
        return if (number < 1) {
            "0 item(s)"
        } else {
            "$number item(s)"
        }
    }

    fun getPriceNumberToString(number: Int): String {
        return if (number < 1) {
            "₦0"
        } else {
            "₦$number"
        }
    }
}